README.md

```diff
@@ -183,6 +183,31 @@ to a command update to the device.
 
 ![General ](https://raw.github.com/dmoranj/iotagent-node-lib/develop/img/iotAgentLib.png "Architecture Overview")
 
+### The TimeInstant attribute
+
+The IoT Agent manages the existence of a special timestamp attribute in the device to entity mapping: **TimeInstant**. This
```

Comment:
The IoT Agent manages a special timestamp attribute as part of the device to entity mapping process